README.md

```diff
@@ -40,8 +40,8 @@ Range, Readings
 
 Establish quality parameters: 
 
-- What is the maximum complexity (CCN) per function? _enter CCN and create corresponding yml in the `.github/workflows` folder
-- How many lines of duplicate code will you tolerate? _enter the number of lines and create corresponding yml in the `.github/workflows` folder
+- What is the maximum complexity (CCN) per function? 3 CCN and create corresponding yml in the `.github/workflows` folder
+- How many lines of duplicate code will you tolerate? 2 the number of lines and create corresponding yml in the `.github/workflows` folder
```

Comment:
But you have `--min-lines 10` in the yml file?

Reply:
corrected to 2